### ISSUE TITLE
ERROR on build_renewable_profiles (technology=offwind-dc) - ON Windows PC

### ISSUE THREAD

**Issue by swagatalpha3** [PyPSA/pypsa-eur#250]
## Checklist

- [x] I am using the current [`master`](https://github.com/PyPSA/pypsa-eur/tree/master) branch or the latest [release](https://github.com/PyPSA/pypsa-eur/releases). Please indicate.
- [ ] I am running on an up-to-date [`pypsa-eur` environment](https://github.com/PyPSA/pypsa-eur/blob/master/envs/environment.yaml). Update via `conda env update -f envs/environment.yaml`.

## Describe the Bug
While running: snakemake -j 1 -n  results/networks/elec_s_128_ec_lcopt_Co2L-24H.nc  --keep-target-files, error occurred in build_renewable_profiles (technology=offwind-dc).

OS > Windows 10 HOME, Os Build 19042.985
------------------------------------------------------------------------------------------------------------
*Note-The similar bug is already reported for Mac OS and still unresolved*
Link to the unresolved bug-->https://github.com/PyPSA/pypsa-eur/issues/249#issue-911538122
------------------------------------------------------------------------------------------------------------
## Error Message
```
[Mon Jun  7 13:09:40 2021]
rule build_renewable_profiles:
    input: networks/base.nc, data/bundle/corine/g250_clc06_V18_5.tif, resources/natura.tiff, data/bundle/GEBCO_2014_2D.nc, resources/country_shapes.geojson, resources/offshore_shapes.geojson, resources/regions_offshore.geojson, cutouts/europe-2013-era5.nc
    output: resources/profile_offwind-dc.nc
    log: logs/build_renewable_profile_offwind-dc.log
    jobid: 14
    benchmark: benchmarks/build_renewable_profiles_offwind-dc
    wildcards: technology=offwind-dc
    resources: mem=20000

Traceback (most recent call last):
Traceback (most recent call last):
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa_eur_1\pypsa-eur\.snakemake\scripts\tmpvuoz9g_j.build_renewable_profiles.py", line 261, in <module>
  File "<string>", line 1, in <module>
    availability = cutout.availabilitymatrix(regions, excluder, **kwargs)
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\multiprocessing\spawn.py", line 116, in spawn_main
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\site-packages\atlite\gis.py", line 520, in compute_availabilitymatrix
    exitcode = _main(fd, parent_sentinel)
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\multiprocessing\spawn.py", line 126, in _main
    self = reduction.pickle.load(from_parent)
EOFError: Ran out of input
    with mp.Pool(**kwargs) as pool:
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\multiprocessing\context.py", line 119, in Pool
    return Pool(processes, initializer, initargs, maxtasksperchild,
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\multiprocessing\pool.py", line 212, in __init__
    self._repopulate_pool()
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\multiprocessing\pool.py", line 303, in _repopulate_pool
    return self._repopulate_pool_static(self._ctx, self.Process,
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\multiprocessing\pool.py", line 326, in _repopulate_pool_static
    w.start()
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\multiprocessing\process.py", line 121, in start
    self._popen = self._Popen(self)
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\multiprocessing\context.py", line 327, in _Popen
    return Popen(process_obj)
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\multiprocessing\popen_spawn_win32.py", line 93, in __init__
    reduction.dump(process_obj, to_child)
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\multiprocessing\reduction.py", line 60, in dump
    ForkingPickler(file, protocol).dump(obj)
_pickle.PicklingError: Can't pickle <function <lambda> at 0x000002315238F9D0>: attribute lookup <lambda> on __main__ failed
[Mon Jun  7 13:09:55 2021]
Error in rule build_renewable_profiles:
    jobid: 14
    output: resources/profile_offwind-dc.nc
    log: logs/build_renewable_profile_offwind-dc.log (check log file(s) for error message)

RuleException:
CalledProcessError in line 197 of D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa_eur_1\pypsa-eur\Snakefile:
Command 'D:/h_da/Lecturs/masterarbeit/pypsaEur/pypsa-eur/envs/eurEnv1/python.exe "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa_eur_1\pypsa-eur\.snakemake\scripts\tmpvuoz9g_j.build_renewable_profiles.py"' returned non-zero exit status 1.
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\site-packages\snakemake\executors\__init__.py", line 2293, in run_wrapper
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa_eur_1\pypsa-eur\Snakefile", line 197, in __rule_build_renewable_profiles
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\site-packages\snakemake\executors\__init__.py", line 569, in _callback
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\concurrent\futures\thread.py", line 57, in run
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\site-packages\snakemake\executors\__init__.py", line 555, in cached_or_run
  File "D:\h_da\Lecturs\masterarbeit\pypsaEur\pypsa-eur\envs\eurEnv1\lib\site-packages\snakemake\executors\__init__.py", line 2381, in run_wrapper
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message

## Python 3.8.10, Package list
#
# Name                    Version                   Build  Channel
affine                    2.3.0                      py_0    conda-forge
aiohttp                   3.7.4            py38h294d835_0    conda-forge
amply                     0.1.4                      py_0    conda-forge
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
async-timeout             3.0.1                   py_1000    conda-forge
atlite                    0.2.3              pyhd8ed1ab_1    conda-forge
attrs                     21.2.0             pyhd8ed1ab_0    conda-forge
azure-common              1.1.27             pyhd8ed1ab_0    conda-forge
azure-core                1.14.0             pyhd8ed1ab_0    conda-forge
azure-storage-blob        12.8.1             pyhd8ed1ab_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
beautifulsoup4            4.9.3              pyhb0f4dca_0    conda-forge
blinker                   1.4                        py_1    conda-forge
blosc                     1.21.0               h0e60522_0    conda-forge
bokeh                     2.3.2            py38haa244fe_0    conda-forge
boost-cpp                 1.74.0               h54f0996_3    conda-forge
boto                      2.49.0                     py_0    conda-forge
boto3                     1.17.78            pyhd8ed1ab_0    conda-forge
botocore                  1.20.78            pyhd8ed1ab_0    conda-forge
brotlipy                  0.7.0           py38h294d835_1001    conda-forge
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2021.5.30            h5b45459_0    conda-forge
cachetools                4.2.2              pyhd8ed1ab_0    conda-forge
cairo                     1.16.0            hb19e0ff_1008    conda-forge
cartopy                   0.19.0.post1     py38hd4bff75_0    conda-forge
cdsapi                    0.5.1              pyhd8ed1ab_0    conda-forge
certifi                   2021.5.30        py38haa244fe_0    conda-forge
cffi                      1.14.5           py38hd8c33c5_0    conda-forge
cfitsio                   3.470                h0af3d06_7    conda-forge
cftime                    1.5.0            py38hcb5f9af_0    conda-forge
chardet                   4.0.0            py38haa244fe_1    conda-forge
click                     7.1.2              pyh9f0ad1d_0    conda-forge
click-plugins             1.1.1                      py_0    conda-forge
cligj                     0.7.1              pyhd8ed1ab_0    conda-forge
cloudpickle               1.6.0                      py_0    conda-forge
colorama                  0.4.4              pyh9f0ad1d_0    conda-forge
conda                     4.10.1           py38haa244fe_0    conda-forge
conda-package-handling    1.7.3            py38h31c79cd_0    conda-forge
configargparse            1.4.1              pyhd8ed1ab_0    conda-forge
connection_pool           0.0.3              pyhd3deb0d_0    conda-forge
countrycode               0.2                      pypi_0    pypi
cryptography              3.4.7            py38hd7da0ea_0    conda-forge
curl                      7.76.1               h789b8ee_2    conda-forge
cycler                    0.10.0                     py_2    conda-forge
cytoolz                   0.11.0           py38h294d835_3    conda-forge
dask                      2021.3.1           pyhd8ed1ab_0    conda-forge
dask-core                 2021.3.1           pyhd8ed1ab_0    conda-forge
datrie                    0.8.2            py38h294d835_2    conda-forge
decorator                 5.0.9              pyhd8ed1ab_0    conda-forge
descartes                 1.1.0                      py_4    conda-forge
distributed               2021.4.1         py38haa244fe_1    conda-forge
docutils                  0.17.1           py38haa244fe_0    conda-forge
entsoe-py                 0.3.2              pyhd8ed1ab_0    conda-forge
et_xmlfile                1.0.1                   py_1001    conda-forge
expat                     2.3.0                h39d44d4_0    conda-forge
filelock                  3.0.12             pyh9f0ad1d_0    conda-forge
fiona                     1.8.18           py38h91b197a_1    conda-forge
fontconfig                2.13.1            h1989441_1005    conda-forge
freetype                  2.10.4               h546665d_1    conda-forge
freexl                    1.0.6                ha8e266a_0    conda-forge
fsspec                    2021.5.0           pyhd8ed1ab_0    conda-forge
gdal                      3.2.1            py38hb9e1232_7    conda-forge
geographiclib             1.50                       py_0    conda-forge
geopandas                 0.9.0              pyhd8ed1ab_0    conda-forge
geopy                     2.1.0              pyhd3deb0d_0    conda-forge
geos                      3.9.1                h39d44d4_2    conda-forge
geotiff                   1.6.0                hee96dd5_4    conda-forge
gettext                   0.19.8.1          h1a89ca6_1005    conda-forge
gitdb                     4.0.7              pyhd8ed1ab_0    conda-forge
gitpython                 3.1.17             pyhd8ed1ab_0    conda-forge
glpk                      4.65              h8ffe710_1004    conda-forge
google-api-core           1.26.3             pyhd8ed1ab_0    conda-forge
google-auth               1.30.0             pyh44b312d_0    conda-forge
google-cloud-core         1.5.0              pyhd3deb0d_0    conda-forge
google-cloud-storage      1.38.0             pyh44b312d_0    conda-forge
google-crc32c             1.1.2            py38h554a69a_0    conda-forge
google-resumable-media    1.2.0              pyhd3deb0d_0    conda-forge
googleapis-common-protos  1.53.0           py38h5b57dd5_0    conda-forge
grpcio                    1.38.0           py38he5377a8_0    conda-forge
gurobi                    9.1.2                    py38_0    gurobi
hdf4                      4.2.15               h0e5069d_3    conda-forge
hdf5                      1.10.6          nompi_h5268f04_1114    conda-forge
heapdict                  1.0.1                      py_0    conda-forge
icu                       68.1                 h0e60522_0    conda-forge
idna                      2.10               pyh9f0ad1d_0    conda-forge
importlib-metadata        4.0.1            py38haa244fe_0    conda-forge
intel-openmp              2021.2.0           h57928b3_616    conda-forge
ipopt                     3.13.4               hf6be2e5_0    conda-forge
ipykernel                 5.3.4            py38h5ca1d4c_0    anaconda
ipython                   7.23.1           py38h43734a8_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
isodate                   0.6.0                      py_1    conda-forge
jdcal                     1.4.1                      py_0    conda-forge
jedi                      0.18.0           py38haa244fe_2    conda-forge
jinja2                    3.0.1              pyhd8ed1ab_0    conda-forge
jmespath                  0.10.0             pyh9f0ad1d_0    conda-forge
joblib                    1.0.1              pyhd8ed1ab_0    conda-forge
jpeg                      9d                   h8ffe710_0    conda-forge
jsonschema                3.2.0              pyhd8ed1ab_3    conda-forge
jupyter_client            6.1.7                      py_0    anaconda
jupyter_core              4.7.1            py38haa244fe_0    conda-forge
kealib                    1.4.14               h96bfa42_2    conda-forge
kiwisolver                1.3.1            py38hbd9d945_1    conda-forge
krb5                      1.19.1               hbae68bd_0    conda-forge
lcms2                     2.12                 h2a16943_0    conda-forge
libarchive                3.5.1                h8686738_1    conda-forge
libblas                   3.9.0                     9_mkl    conda-forge
libcblas                  3.9.0                     9_mkl    conda-forge
libcrc32c                 1.1.1                h0e60522_2    conda-forge
libcurl                   7.76.1               h789b8ee_2    conda-forge
libffi                    3.3                  h0e60522_2    conda-forge
libflang                  5.0.0           h6538335_20180525    conda-forge
libgdal                   3.2.1                hca2c63a_7    conda-forge
libglib                   2.68.2               h1e62bf3_0    conda-forge
libiconv                  1.16                 he774522_0    conda-forge
libkml                    1.3.0             h9859afa_1013    conda-forge
liblapack                 3.9.0                     9_mkl    conda-forge
libnetcdf                 4.7.4           nompi_h3a9aa94_107    conda-forge
libpng                    1.6.37               h1d00b33_2    conda-forge
libpq                     13.3                 hfcc5ef8_0    conda-forge
libprotobuf               3.17.0               h7755175_0    conda-forge
librttopo                 1.1.0                hb340de5_6    conda-forge
libsodium                 1.0.18               h62dcd97_0    anaconda
libsolv                   0.7.18               h7755175_0    conda-forge
libspatialindex           1.9.3                h39d44d4_3    conda-forge
libspatialite             5.0.1                h987baeb_3    conda-forge
libssh2                   1.9.0                h680486a_6    conda-forge
libtiff                   4.2.0                h763f289_2    conda-forge
libwebp-base              1.2.0                h8ffe710_2    conda-forge
libxml2                   2.9.12               hf5bbc77_0    conda-forge
libxslt                   1.1.33               h65864e5_2    conda-forge
llvm-meta                 5.0.0                         0    conda-forge
locket                    0.2.0                      py_2    conda-forge
lxml                      4.6.3            py38h292cb97_0    conda-forge
lz4-c                     1.9.3                h8ffe710_0    conda-forge
lzo                       2.10              he774522_1000    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
mamba                     0.13.0           py38hdd88130_0    conda-forge
markupsafe                2.0.1            py38h294d835_0    conda-forge
matplotlib-base           3.4.2            py38heae8d8c_0    conda-forge
matplotlib-inline         0.1.2              pyhd8ed1ab_2    conda-forge
memory_profiler           0.58.0                     py_0    conda-forge
menuinst                  1.4.16           py38h32f6830_1    conda-forge
metis                     5.1.0             h0e60522_1006    conda-forge
mkl                       2021.2.0           hb70f87d_389    conda-forge
mock                      4.0.3            py38haa244fe_1    conda-forge
msgpack-python            1.0.2            py38hbd9d945_1    conda-forge
msrest                    0.6.21             pyh44b312d_0    conda-forge
msys2-conda-epoch         20160418                      1    conda-forge
multidict                 5.1.0            py38h294d835_1    conda-forge
mumps-seq                 5.2.1               hb3f9cae_10    conda-forge
munch                     2.5.0                      py_0    conda-forge
nbformat                  5.1.3              pyhd8ed1ab_0    conda-forge
netcdf4                   1.5.6           nompi_py38h8504c39_102    conda-forge
networkx                  2.5                        py_0    conda-forge
numexpr                   2.7.3            py38h4c96930_0    conda-forge
numpy                     1.19.0           py38h72c728b_0    conda-forge
oauthlib                  3.0.1                      py_0    conda-forge
olefile                   0.46               pyh9f0ad1d_1    conda-forge
openjpeg                  2.4.0                hb211442_1    conda-forge
openmp                    5.0.0                    vc14_1    conda-forge
openpyxl                  3.0.7              pyhd8ed1ab_0    conda-forge
openssl                   1.1.1k               h8ffe710_0    conda-forge
packaging                 20.9               pyh44b312d_0    conda-forge
pandas                    1.2.4            py38h60cbd38_0    conda-forge
parso                     0.8.2              pyhd8ed1ab_0    conda-forge
partd                     1.2.0              pyhd8ed1ab_0    conda-forge
patsy                     0.5.1                      py_0    conda-forge
pcre                      8.44                 ha925a31_0    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    8.2.0            py38h9273828_1    conda-forge
pip                       21.1.2             pyhd8ed1ab_0    conda-forge
pixman                    0.40.0               h8ffe710_0    conda-forge
ply                       3.11                       py_1    conda-forge
poppler                   0.89.0               h3772339_5    conda-forge
poppler-data              0.4.10                        0    conda-forge
postgresql                13.3                 h1c22c4f_0    conda-forge
powerplantmatching        0.4.8              pyhd8ed1ab_0    conda-forge
progressbar2              3.53.1             pyh9f0ad1d_0    conda-forge
proj                      7.2.0                h1cfcee9_2    conda-forge
prompt-toolkit            3.0.18             pyha770c72_0    conda-forge
protobuf                  3.17.0           py38h885f38d_0    conda-forge
psutil                    5.8.0            py38h294d835_1    conda-forge
pulp                      2.4              py38haa244fe_0    conda-forge
pyasn1                    0.4.8                      py_0    conda-forge
pyasn1-modules            0.2.7                      py_0    conda-forge
pycosat                   0.6.3           py38h294d835_1006    conda-forge
pycountry                 20.7.3             pyh9f0ad1d_0    conda-forge
pycparser                 2.20               pyh9f0ad1d_2    conda-forge
pygments                  2.9.0              pyhd8ed1ab_0    conda-forge
pyjwt                     2.1.0              pyhd8ed1ab_0    conda-forge
pyomo                     6.0              py38h885f38d_0    conda-forge
pyopenssl                 20.0.1             pyhd8ed1ab_0    conda-forge
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pyproj                    3.0.1            py38h0ae89ba_0    conda-forge
pypsa                     0.17.1             pyh9f0ad1d_0    conda-forge
pyrsistent                0.17.3           py38h294d835_2    conda-forge
pyshp                     2.1.3              pyh44b312d_0    conda-forge
pysocks                   1.7.1            py38haa244fe_3    conda-forge
pytables                  3.6.1            py38h153c448_3    conda-forge
python                    3.8.10          h7840368_1_cpython    conda-forge
python-dateutil           2.8.1                      py_0    conda-forge
python-utils              2.5.6              pyh44b312d_0    conda-forge
python_abi                3.8                      1_cp38    conda-forge
pytz                      2021.1             pyhd8ed1ab_0    conda-forge
pywin32                   300              py38h294d835_0    conda-forge
pyyaml                    5.4.1            py38h294d835_0    conda-forge
pyzmq                     19.0.2           py38ha925a31_1    anaconda
rasterio                  1.2.3            py38h4f558c9_0    conda-forge
ratelimiter               1.2.0                   py_1002    conda-forge
reproc                    14.2.1               h8ffe710_0    conda-forge
reproc-cpp                14.2.1               h0e60522_0    conda-forge
requests                  2.25.1             pyhd3deb0d_0    conda-forge
requests-oauthlib         1.3.0              pyh9f0ad1d_0    conda-forge
rsa                       4.7.2              pyh44b312d_0    conda-forge
rtree                     0.9.7            py38h8b54edf_1    conda-forge
ruamel_yaml               0.15.80         py38h294d835_1004    conda-forge
s3transfer                0.4.2              pyhd8ed1ab_0    conda-forge
scikit-learn              0.24.2           py38h5d5d464_0    conda-forge
scipy                     1.6.3            py38he847743_0    conda-forge
seaborn                   0.11.1               hd8ed1ab_1    conda-forge
seaborn-base              0.11.1             pyhd8ed1ab_1    conda-forge
setuptools                49.6.0           py38haa244fe_3    conda-forge
shapely                   1.7.1            py38h13ff51f_4    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
sklearn                   0.0                      pypi_0    pypi
smart_open                5.0.0              pyhd8ed1ab_0    conda-forge
smmap                     3.0.5              pyh44b312d_0    conda-forge
snakemake-minimal         6.4.0              pyhdfd78af_0    bioconda
snuggs                    1.4.7                      py_0    conda-forge
sortedcontainers          2.4.0              pyhd8ed1ab_0    conda-forge
soupsieve                 2.0.1                      py_1    conda-forge
spyder-kernels            2.0.3            py38haa244fe_0    conda-forge
sqlite                    3.35.5               h8ffe710_0    conda-forge
statsmodels               0.12.2           py38h347fdf6_0    conda-forge
stopit                    1.1.2                      py_0    conda-forge
tbb                       2021.2.0             h2d74725_0    conda-forge
tblib                     1.7.0              pyhd8ed1ab_0    conda-forge
threadpoolctl             2.1.0              pyh5ca1d4c_0    conda-forge
tiledb                    2.2.9                hf84e3da_0    conda-forge
tk                        8.6.10               h8ffe710_1    conda-forge
toolz                     0.11.1                     py_0    conda-forge
toposort                  1.6                pyhd8ed1ab_0    conda-forge
tornado                   6.1              py38h294d835_1    conda-forge
tqdm                      4.60.0             pyhd8ed1ab_0    conda-forge
traitlets                 5.0.5                      py_0    conda-forge
tsam                      1.1.0                    pypi_0    pypi
typing-extensions         3.7.4.3                       0    conda-forge
typing_extensions         3.7.4.3                    py_0    conda-forge
urllib3                   1.26.4             pyhd8ed1ab_0    conda-forge
vc                        14.2                 hb210afc_4    conda-forge
vresutils                 0.3.1                    pypi_0    pypi
vs2015_runtime            14.28.29325          h5e1d092_4    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
wheel                     0.36.2             pyhd3deb0d_0    conda-forge
win_inet_pton             1.1.0            py38haa244fe_2    conda-forge
wincertstore              0.2             py38haa244fe_1006    conda-forge
wrapt                     1.12.1           py38h294d835_3    conda-forge
xarray                    0.18.2             pyhd8ed1ab_0    conda-forge
xerces-c                  3.2.3                h0e60522_2    conda-forge
xlrd                      2.0.1              pyhd8ed1ab_3    conda-forge
xz                        5.2.5                h62dcd97_1    conda-forge
yaml                      0.2.5                he774522_0    conda-forge
yarl                      1.5.1            py38h1e8a9f7_0    conda-forge
zeromq                    4.3.2                ha925a31_3    anaconda
zict                      2.0.0                      py_0    conda-forge
zipp                      3.4.1              pyhd8ed1ab_0    conda-forge
zlib                      1.2.11            h62dcd97_1010    conda-forge
zstd                      1.4.9                h6255e5f_0    conda-forge
```

**Comment by euronion** [comment]
Hi @swagatalpha3 ,

Thanks for the report.
In the mailinglist you mentioned specific problems when importing the cutout directly.
Does this problem still _persist_ or is it now only the `multiprocessing` issue?

**Comment by swagatalpha3** [comment]
Hi @euronion,

The cutout problem seems to be solved now. Now I can import the cutout directly.
Yes, apparently the only problem is *multiprocessing* I think.

**Comment by euronion** [comment]
Ok, well if it is only the `multiprocessing` issue above, then it seems to be only a duplicate of #249 .

I'll close here for now. Reopen if the problem persists after #249 has been adressed.

**Comment by euronion** [comment]
Duplicate of #249
